scripts/provisioning/s3_setup

```diff
@@ -195,27 +216,21 @@ configure_openldap()
         /opt/seagate/cortx/s3/install/ldap/replication/setupReplicationScript.sh -h hosts_list_file.txt -p "$rootdnpasswd"
         if [ $? -ne 0 ]
         then
-            echo "ERROR: failed to configure ldap replication, exiting."
+            echo "ERROR: Failed to configure ldap replication, exiting."
+            echo "ERROR: You may want to enable openldap logs, and check what exactly failed."
```

Comment:
this statement not required as slapd logs are enabled by default above

Reply:
done